tests/product.py

```diff
@@ -96,5 +96,23 @@ def test_get_all_products(self):
         self.assertEqual(len(json_response), 3)
 
     # TODO: Delete product
+    def test_delete_a_product(self):
+        """
+        Ensure we can delete a product
+        """
+        # Seed database with a product to delete
+        self.test_create_product()
```

Comment:
The `test_create_product` method is meant to test the create function. The existing Bangazon code also uses that method for seeding the test database also, which is poor practice for a couple of reasons. 1. It violates the Single Responsibility Principle, and 2. it makes the code less 'DRY' (i.e. the code becomes repetitive, sometimes referred to as 'WET' code).

Instead, there should be separate code written specifically for the purpose of seeding the database before each test. The `setUp` method runs before each test in a given test class, which makes it the ideal location for this sort of functionality. 

Remove the `test_create_product` method from your `test_delete_a_product` method. Then, modify your `setUp` method to seed product data into the test db before each test. Ping me on slack once you've pushed up changes, and then you'll be good to go!